brownfield_django/templates/main/team/team_home.html

```diff
@@ -41,10 +41,10 @@
             <li>
               {{ document.name }}
               {% if document.name == "Link: Brownfield Action Reference Site" %}
-                <a class="btn btn-default pull-right" href=http://brownfieldref.ccnmtl.columbia.edu/" target="_blank">View</a>
+                  <a class="btn btn-default pull-right" href=http://brownfieldref.ccnmtl.columbia.edu/" target="_blank">View</a>
```

Comment:
Missing a double-quote here, should be: `href="http://brownfieldref.ccnmtl.columbia.edu/"`